homeassistant/components/risco/binary_sensor.py

```diff
@@ -3,23 +3,76 @@
 from __future__ import annotations
 
 from collections.abc import Mapping
+from itertools import chain
 from typing import Any
 
 from pyrisco.cloud.zone import Zone as CloudZone
+from pyrisco.common import System
 from pyrisco.local.zone import Zone as LocalZone
 
 from homeassistant.components.binary_sensor import (
     BinarySensorDeviceClass,
     BinarySensorEntity,
+    BinarySensorEntityDescription,
 )
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import LocalData, RiscoDataUpdateCoordinator, is_local
-from .const import DATA_COORDINATOR, DOMAIN
+from .const import DATA_COORDINATOR, DOMAIN, SYSTEM_UPDATE_SIGNAL
 from .entity import RiscoCloudZoneEntity, RiscoLocalZoneEntity
 
+SYSTEM_ENTITY_DESCRIPTIONS = [
+    BinarySensorEntityDescription(
+        key="low_battery_trouble",
+        name="Low battery trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
```

Comment:
Would the battery low device class fit?

Reply:
From what I understand this means a general problem with the battery, either it's empty, or defective, or not working for any other reason. Not sure if that fits the Battery class, but happy to change if it does.

Reply:
I think keeping it this way best reflects the risco use case 👍

Reply:
After looking at a couple of Risco docs, this is usually described as low/no battery, so I'm going with your suggestion